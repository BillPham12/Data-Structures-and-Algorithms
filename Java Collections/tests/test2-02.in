my
name
is
kong
krap
ku
ta
li
si
make
fo
k
u
c
k
