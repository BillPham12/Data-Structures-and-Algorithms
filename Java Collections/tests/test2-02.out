my
is
krap
ta
si
fo
u
k
name
kong
ku
li
make
k
c
